latex/chain.tex

```diff
@@ -97,7 +97,7 @@ \subsection{Verifiable Random Functions (VRF)}
   %
   \emph{Derived Types}
   \begin{align*}
-    \PoolDistr = \VKey \mapsto [0, 1] \text{ \hspace{3cm}stake pool distribution}
+    \PoolDistr = \HashKey \mapsto Coin \text{ \hspace{3cm}stake pool distribution}
```

Comment:
There is indeed a type error here, thanks!

I think we may want to use `HashKey |-> [0, 1]`.  When we use the value in the `VRF` rule (fig 56), the sigma needs to be a relative stake. and this needs to be relative to the total stake as it was at the beginning of the epoch.

So I'd propose: calculating `pd'` in equation 21 of figure 48 as relative stake by dividing all the values by the total coin in `stake`.  And thin in in the `VRF` rule we also need to convert the `VKey` to a `HashKey`, since `bissuer` returns a `VKey`.